person.rb

```diff
@@ -12,6 +13,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(date, book)
+    Rental.new(date, book, self)
```

Comment:
- You have done well so far, but please note that in future milestones, this code might give you an error because your `Rental` class takes in `book, person, date` in that order. Please fix this 🙏